hooks/pre_validations/31-mongo_storage_engine.rb

```diff
@@ -0,0 +1,54 @@
+MONGO_DIR = '/var/lib/mongodb/'
+scenario = YAML.load_file('./config/last_scenario.yaml').fetch(:answer_file)
+
+# Pull MongoDB connection info from answer file
+if scenario = 'katello'
+  MONGO_HOST = param('katello', 'pulp_db_seeds').value
+else
+  MONGO_HOST = param('foreman_proxy_content', 'pulp_db_seeds').value
+end
+
+def fail_and_exit(message)
```

Comment:
Would this be useful in https://github.com/Katello/katello-installer/blob/master/hooks/boot/01-helpers.rb ?

Reply:
I do not understand objects and classes so for now I will just keep it here, after this is merged it can be changed around if you want.